hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/JournalSet.java

```diff
@@ -40,10 +40,12 @@
 import org.apache.hadoop.hdfs.server.protocol.NamespaceInfo;
 import org.apache.hadoop.hdfs.server.protocol.RemoteEditLog;
 import org.apache.hadoop.hdfs.server.protocol.RemoteEditLogManifest;
-import org.apache.hadoop.thirdparty.com.google.common.annotations.VisibleForTesting;
+import org.apache.hadoop.util.Sets;
+
+import org.apache.hadoop.thirdparty.com.google.common.annotations
+    .VisibleForTesting;
```

Comment:
We don't need to break lines here since the 80-length rule does not apply to import statements.